File: Problem_solving/Home_work/matrix_mult.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>

#define row 1000
#define col 1000

int arr[row][col], brr[row][col], crr[row][col];


int main()
{ 
	int i = 0, j = 0, k = 0;

	for(i = 0; i < row; i++)
	{
		for(j = 0; j < col; j++)
		{
			arr[i][j] = (int)rand() % 100;
			brr[i][j] = (int)rand() % 97;
		}
	}

	for(i = 0; i < row; i++)
	{
		for(j = 0; j < col; j++)
		{
			for(k = 0; k < row; k++)
			{
				crr[i][j] += (arr[k][i] * brr[j][k]);
			}
		}
	}

	return 0;
}

File: Problem_solving/Home_work/matrix_mult.py
import random

row = 3
col = 3
a = [[0]*col]*row
b = [[0]*col]*row
c = [[0]*col]*row
for i in range(0, row):
    for j in range(0, row):
        a[i][j] = random.randint(0,9)
        b[i][j] = random.randint(0,9)


for i in range(0, row):
    for j in range(0, col):
        c[i][j] = 0
        for k in range (0, row):
            c[i][j] += a[k][i] * b[j][k]

print(c)